lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -1114,6 +1113,40 @@ def view_init(self, elev=None, azim=None, roll=None, vertical_axis="z",
         -YZ          0     180
         ==========   ====  ====
 
+        A roll angle of 0, 90, 180, or 270 degrees will rotate these views
+        while keeping the axes horizontal or vertical.
+
+        The *azim*, *elev*, *roll* angles correspond to rotations of the scene
+        observed by a stationary camera, as follows (assuming a default vertical
+        axis of 'z'). First, a left-handed rotation about the z axis is applied
+        (*azim*), then a right-handed rotation about the (camera) y axis (*elev*),
+        then a right-handed rotation about the (camera) x axis (*roll*). Here,
+        the z, y, and x axis are fixed axes (not the axes that rotate together
+        with the original scene).
+
+        If you would like to make the connection with quaternions (because
+        `Euler angles are horrible
```

Comment:
This is an excellent reference, but I think the tone should be more neural. E.g. "See X for a discussion on why quaternions are a more natural way to represent orientations)

Reply:
It is being changed.

Reply:
> It is being changed.

What do you mean by this? I don’t see any changes here. Did you forget to push?

Reply:
Oh I'm sorry, it is less than clear, I meant to say: I'm changing it and I'll push a commit once I've tried to address the other issues raised too. (It may take a little while.)